pyiron/lammps/base.py

```diff
@@ -912,6 +912,12 @@ def collect_dump_file(self, file_name="dump.out", cwd=None):
         )
         output["forces"] = np.matmul(forces, rotation_lammps2orig)
 
+        if "vx" in content[0].columns.values and "vy" in content[0].columns.values and "vz" in content[0].columns.values:
```

Comment:
I find `if np.all([flag in content[0].columns.values for flag in ["vx", "vy", "vz"]]):` easier to read. If you agree or are neutral, please change. If you passionately disagree I'm content to live with it as is.